Readme.md

```diff
@@ -1,7 +1,7 @@
 Eslint Config Calvium
 =====================
 
-Current React Native tested versions from `52.x` but it may work in earliers too.
+Current React Native tested versions from `60.x` but it may work in earliers too.
```

Comment:
typo (not introduced here) - `it may work in earlier versions too`

Reply:
fixed